### PR TITLE
I have added meta tags to h1 files. Made some h2 files h1 files so th…

### PR DESCRIPTION
**Pull request by patrickmcgreevy**
…at they will be indexed as diferent topics. There is a navigation bar that links to parts of the page which should help our link

count. The css for the navigation bar is in the public/assets/css/main.css file. Let me know if there are any other changes you would like to see.

### CODE REVIEW

**Comment by InternetExplorer7** [comment]
You are using an older version of the website which is probably why there are merge conflicts. Please merge with the latest commit in master and reapply changes. Also, move all meta tags to head.

**Comment by patrickmcgreevy** [comment]
I merged with master. My repo is identical to master aside from the changes I've made. i think that any conflicts are resolved.

File: public/index.html [hunk 1]
```diff
@@ -34,9 +50,11 @@ <h1 style="font-family: Avenir;"><img width="250px" src="http://www.kavehkhorr.a
 
 				<!-- Two -->
 					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in" id="first">
-						<div class="content">
-							<h2>You + 149 other hackers.</h2>
-							<p>Join us on January 12-13, 2019 for an exciting weekend of hacking in Seattle, WA. Decisions will be released by early December.</p>
+
+						<div id="hackers"class="content">
+							<h1>You + 149 other hackers.</h1>
+							<p>Join us on January 19-20, 2019 for an exciting weekend of hacking in the Seattle Space Needle. Decisions will be released by early December.</p>
```
Comment (InternetExplorer7) [inline public/index.html]: January 12-13, 2019.

**Review by InternetExplorer7 — CHANGES_REQUESTED** [review]
There are a few other instances of incorrect dates.

File: public/index.html [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
+        <meta name="Hack Washington Sponsor Logos" content="The logos of and info about each Hack Washington @ The Seattle Space Needle sponsor"/>
+
 		<title>Hack//WA</title>
     	<link rel="icon" type="image/png" href="http://www.kavehkhorr.am/hackwashington/website_stuf/favicon.jpg" sizes="50x50" />
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
 		<link rel="stylesheet" href="assets/css/main.css" />
 		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
 	</head>
+<nav>
```
Comment (InternetExplorer7) [inline public/index.html:25]: Is this a visual change? If so, what's the visual change here?

**Review by InternetExplorer7 — CHANGES_REQUESTED** [review]
Are there visual changes?

File: public/index.html (hunk 2 above)
Comment (patrickmcgreevy) [reply public/index.html:25]: I added a navigation bar to the top of the page that allows visitors to jump to each section of the site.

File: public/index.html (hunk 1 above)
Comment (patrickmcgreevy) [reply public/index.html]: Fixed them.

File: public/index.html (hunk 2 above)
Comment (InternetExplorer7) [reply public/index.html:25]: What does that visual change look like?

Comment (InternetExplorer7) [reply public/index.html:25]: Attach a screenshot, please.

Comment (patrickmcgreevy) [reply public/index.html:25]: hackwanavbarss

Comment (patrickmcgreevy) [reply public/index.html:25]: The bar doesn't follow the user down the page. Meaning that the user can scroll past it.

**Review by InternetExplorer7 — APPROVED** [review]

**Pull request merged**